src/Components/GameBoard/GameBoard.js

```diff
@@ -0,0 +1,54 @@
+import UserGuessRow from '../UserGuessRow/UserGuessRow';
+import './GameBoard.css';
+
+const GameBoard = ({ typedLetters, currentWordInPlay, submittedWords }) => {
+  const generateRows = () => {
+    const rows = [];
+
+    const pastRows = submittedWords.map((word, index) => {
+      return (
+        <UserGuessRow 
+          className='past-row'
+          typedLetters={word.split('')}
+          currentWordInPlay={currentWordInPlay.split('')}
+          key={index}
+        />
+      );
+    });
+    rows.push(...pastRows);
+
+    if (rows.length < 6) {
+      const activeRow = (
+        <UserGuessRow 
+          className="active-row"
+          typedLetters={typedLetters}
+          key={rows.length}
+        />
+      );
+      rows.push(activeRow);
+    }
+    
+    while (rows.length < 6) {
+      const futureRow = (
+        <UserGuessRow
+          className="future-row"
+          typedLetters={["", "", "", "", ""]}
+          key={rows.length}
+        />
+      );
+      rows.push(futureRow);
+    }
```

Comment:
This is really interesting!  It looks like you use past words to build the board? 

I had to take a short break from the code review to actually go and play the game 🤣

And now I am super impressed that you chose to build this for your project!  I have been seeing tons of logic, and it's awesome that you're putting this together in a short period of time!  I wouldn't have tried to tackle something like this, as it is now giving me flashbacks to doing RockPaperScissors in Mod1...